entry/src/main/config.json

```diff
@@ -56,14 +56,37 @@
         },
         "pages": [
           "pages/index",
-          "pages/CleanUtilsSample"
+          "pages/CleanUtilsSample",
+          "pages/LocationUtilsSample"
         ],
         "name": ".MainAbility",
         "window": {
           "designWidth": 720,
           "autoDesignWidth": false
         }
       }
+    ],
+    "reqPermissions": [
+      {
```

Comment:
elaborate the location permission details like in this sample
https://gitee.com/harmonyos/samples/blob/master/device/Location/entry/src/main/config.json